semgrep/semgrep/cli.py

```diff
@@ -422,6 +414,17 @@ def cli() -> None:
         help=argparse.SUPPRESS,
         # help="Deprecated alias for --json + --debug",
     )
+    output.add_argument(
+        "--save-test-output-tar",
```

Comment:
This argument is only used internally. Let's suppress it so it doesn't clutter up the help text, and so external users don't inadvertently start relying on it.